doc/source/templates/05_dreambooth_finetuning/dreambooth/flags.py

```diff
@@ -146,5 +149,10 @@ def run_model_flags():
             "Enable using Ray Data to use multiple GPU workers to perform inference."
         ),
     )
+    parser.add_argument(
+        "--lora_weights_dir",
```

Comment:
If this is not passed in, can we assume it's the same as `model_dir`? I guess my question is that do we need this extra parameter? instead of re-using model_dir